py/runtime.c

```diff
@@ -1326,17 +1326,24 @@ mp_obj_t mp_make_raise_obj(mp_obj_t o) {
 
 mp_obj_t mp_import_name(qstr name, mp_obj_t fromlist, mp_obj_t level) {
     DEBUG_printf("import name '%s' level=%d\n", qstr_str(name), MP_OBJ_SMALL_INT_VALUE(level));
-
     // build args array
-    mp_obj_t args[5];
-    args[0] = MP_OBJ_NEW_QSTR(name);
-    args[1] = mp_const_none; // TODO should be globals
-    args[2] = mp_const_none; // TODO should be locals
-    args[3] = fromlist;
-    args[4] = level;
-
-    // TODO lookup __import__ and call that instead of going straight to builtin implementation
-    return mp_builtin___import__(5, args);
+    mp_obj_t argv[5];
```

Comment:
Is there a reason to rename this array?  Best to keep it unchanged to minimise the diff.